meshapp/src/components/Header.tsx

```diff
@@ -1,30 +1,13 @@
-import { AppBar, Grid, IconButton, Toolbar, Typography } from "@mui/material";
-
-import MenuIcon from "@mui/icons-material/TableRows";
-import NotificationsIcon from "@mui/icons-material/Notifications";
-
 import React from "react";
+import { AppBar, Grid, Toolbar, Typography } from "@mui/material";
+import SideMenu from "./SideMenu";
```

Comment:
This import does not appear to be working. Looks like you have a separate Components folder where this side menu is located.